app/services/cocina_description_support.rb

```diff
@@ -27,6 +27,19 @@ def self.title(title:)
     [{ value: title }]
   end
 
+  def self.contact_emails(contact_emails:)
+    contact_emails.map do |contact_email|
+      contact_email = contact_email.attributes if contact_email.respond_to?(:attributes)
```

Comment:
Why is this necessary? (Can you put explanation in a code comment.)

Reply:
Hmm, this basically directly from the related_links/related_works nested components. I'll see if I can determine if it's necessary or not.

Reply:
done.